app/wasm_config.go

```diff
@@ -1,25 +0,0 @@
-package app
-
-import (
-	wasmkeeper "github.com/CosmWasm/wasmd/x/wasm/keeper"
-)
-
-const (
-	// DefaultandromedaInstanceCost is initially set the same as in wasmd
-	DefaultandromedaInstanceCost uint64 = 60_000
-	// DefaultandromedaCompileCost set to a large number for testing
-	DefaultandromedaCompileCost uint64 = 100
-)
-
-// andromedaGasRegisterConfig is defaults plus a custom compile amount
-func andromedaGasRegisterConfig() wasmkeeper.WasmGasRegisterConfig {
-	gasConfig := wasmkeeper.DefaultGasRegisterConfig()
```

Comment:
juno did this from the very early days pre alpha/beta - no longer required since SDK v45 as this is the default compute cost for exewcution